inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -1156,4 +1157,38 @@ std::shared_ptr<ngraph::Node> V10Parser::LayerCreator<ngraph::op::v1::LogicalNot
     return std::make_shared<ngraph::op::v1::LogicalNot>(inputs[0]);
 }
 
+// Constant layer
+template <>
+std::shared_ptr<ngraph::Node> V10Parser::LayerCreator<ngraph::op::v0::Constant>::createLayer(
```

Comment:
Recently we removed LayerCreator for the constant operation. I propose to don't add the new layer creator. At the current moment we are removing this functional. 

We create a Constant op using the Visitor API (visit_attributes method).

Reply:
@ilyachur i tried to use attribute visitor but problem is - on visiting attribute constant do create empty buffer, second i can't set  SharedBuffer from on_attribute  as there is no way to do that. Tried to use DirectValueAccessor, but  in this case i need to remove buffer initialization from Constant on visitor method, this will lead to problems with constant initialization from another parts of code thru visitors.

What is the problem with layer creator? Attribute visitor will  be used for serializer, creator for deserializer.